<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaunchMethodSelectionWindowClass</class>
 <widget class="QDialog" name="LaunchMethodSelectionWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Launch method preselection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>The launch method selected here will be preselected when creating a new flight. It is still possible to select a different launch method.
</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="preselectionCheckbox">
     <property name="title">
      <string>&amp;Activate preselection</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="SkComboBox" name="launchMethodInput"/>
      </item>
      <item>
       <widget class="QCheckBox" name="loadPreselectionCheckbox">
        <property name="text">
         <string>Save preselection on application exit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SkComboBox</class>
   <extends>QComboBox</extends>
   <header>src/gui/widgets/SkComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>preselectionCheckbox</tabstop>
  <tabstop>launchMethodInput</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LaunchMethodSelectionWindowClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LaunchMethodSelectionWindowClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
